elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -227,42 +223,25 @@ $variables: (
   --pfe-cta--Color--focus:              #{pfe-color(ui-complement)};
 }
 
-
 /// ===========================================================================
 /// PRIMARY CTAs
 /// ===========================================================================
 :host([pfe-priority="primary"]) {
-  // Button padding and border radius settings
-  --pfe-cta--BorderRadius:              #{pfe-var(ui--border-radius)};
+  --pfe-cta--main:        #{pfe-color(ui-accent)};
```

Comment:
The broadcast branch did away with the main concept so I think it would be worth pulling the latest of this file and confirming if these changes (from this point and below) are still needed.  I suspect they are not.